src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
-}
+  const basePrice = days * 40;
+  const mediumTerm = 3;
+  const longTerm = 7;
+  const mediumTermDiscount = 20;
+  const longTermDiscount = 50;
+
+  if (days < mediumTerm) {
+    return basePrice;
+  }
 
+  if (days >= mediumTerm && days < longTerm) {
+    return basePrice - mediumTermDiscount;
+  }
```

Comment:
There's no need for the second condition `days < longTerm` since the first `if` statement already returns for days less than `mediumTerm`. You can remove it to simplify the code.